src/components/Person.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Person ({name, age = null, sex, isMarried = false, partnerName = null}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
```

Comment:
Вынеси проверку на существование age за предела параграфа, он не должен отображаться в разметке, если нет возраста
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```